R/conversions.R

```diff
@@ -73,10 +73,14 @@ convertDGEobj <- function(dgeObj, Class) {
 
     supportedClasses <- list("RANGEDSUMMARIZEDEXPERIMENT", "RSE",
                              "EXPRESSIONSET", "ES", "LIST")
-    assert_that(!missing(dgeObj),
-                !missing(Class),
-                class(Class) == "character",
-                toupper(Class) %in% supportedClasses)
+
+    assertthat::assert_that(!missing(dgeObj),
+                            !missing(Class),
+                            msg = "Specify both a DGEobj and a desired Class to convert the DGEobj to.")
+    assertthat::assert_that(class(Class) == "character",
+                            msg = "Specify the Class via character string" )
+    assertthat::assert_that(toupper(Class) %in% supportedClasses,
+                            msg = 'The specified class must be one of "RangedSummarizedExperiment", "RSE", "ExpressionSet", or "ES".')
```

Comment:
Let's incorporate the supportedClasses variable here, this way the message will always match.  ie paste/collapse the values

Reply:
The `supportedClasses` variable is in all caps (so "RANGEDSUMMARIZEDEXPERIMENT", etc.) and converting back to correct cases will be challenging, as some are title and some are all caps. I'm not sure of a good way to bring it back in.

Reply:
ok, no problem, we will likely drop the conversions also.